qa/python_models/dlpack_test/model.py

```diff
@@ -56,6 +56,23 @@ def test_pytorch_dlpack(self):
             self.assertTrue(
                 pytorch_tensor.type() == pytorch_tensor_dlpack.type())
 
+            # Now let's check that upgraded DLPack implementation also
+            # works as expected, i.e. from_dlpack should work with 
+            # external pytorch tensor directly
+
+            pb_tensor_upgraded = pb_utils.Tensor.from_dlpack('test_tensor',
+                                                    pytorch_tensor)
+            self.assertTrue(
+                np.all(pb_tensor_upgraded.as_numpy() == pytorch_tensor.numpy()))
+
+            # Here we check that `pb_tensor` as a producer, properly
+            # invokes `__dlpack__` and `__dlpack_device__`
+            pytorch_tensor_dlpack = from_dlpack(pb_tensor_upgraded)
+            self.assertTrue(torch.all(pytorch_tensor_dlpack == pytorch_tensor))
```

Comment:
Are you able to make this a bit more easy-to-read with something like `self.assertTrue(torch.equal(pytorch_tensor_dlpack, pytorch_tensor))`? This also should skip the computation if the tensors are not the same shape.